sktime/forecasting/base/adapters/_fbprophet.py

```diff
@@ -35,6 +36,7 @@ def fit(self, y, X=None, fh=None, **fit_params):
         self._instantiate_model()
         self._check_changepoints()
         self._set_y_X(y, X, enforce_index_type=pd.DatetimeIndex)
+        y, X = check_y_X(y, X)
```

Comment:
This happens inside `self._set_y_X(...)` so not necessary here or any of the other files where this is added in this PR.